DEVELOPMENT.md

```diff
@@ -222,7 +226,8 @@ ko delete --ignore-not-found=true \
   -f config/monitoring/100-common \
   -f config/ \
   -f ./third_party/config/build/release.yaml \
-  -f ./third_party/istio-1.0.2/istio.yaml
+  -f ./third_party/istio-1.0.2/istio.yaml \
+  -f ./third_party/istio-1.0.2/istio-crds.yaml
```

Comment:
please reorder these two lines so `istio-crds.yaml` come first?

Reply:
I purposefully ordered the cleanup that way because we want to cleanup in the reverse order that we deployed. In other words, delete the instances of the Istio CRDs before removing the actual CRD definitions.